kernel/public/test-scenes/0.101.move-inside-scene/scene.json

```diff
@@ -0,0 +1,48 @@
+{
+  "display": {
+    "title": "interactive-text",
+    "favicon": "favicon_asset"
+  },
+  "contact": {
+    "name": "Ezequiel",
+    "email": "ezequiel@decentraland.org"
```

Comment:
It's not necessary for you to define this. It's ok if you want to leave this like it is, just wanted to mention it :)